routers/api/actions/runner/runner.go

```diff
@@ -145,6 +145,22 @@ func (s *Service) UpdateTask(
 		return nil, status.Errorf(codes.Internal, "update task: %v", err)
 	}
 
+	for k, v := range req.Msg.Outputs {
+		if len(k) > 255 {
+			log.Warn("Ignore the output of task %d because the key is too long: %q", task.ID, k)
+			continue
+		}
+		if err := actions_model.InsertTaskOutputIfNotExist(ctx, task.ID, k, v); err != nil {
```

Comment:
Is there any limitation for the output value?

Reply:
Since the output value type is `TEXT`, I think it's okay to let it fail into a database operation error when the value is too long.

Reply:
What's the expected max length? 1M?

Reply:
TBH, I don't know the exact number and I couldn't find any documentation on GitHub to define it. However, since the way to write values is `run: echo "key=value" >> "$GITHUB_OUTPUT"`, the length of values should be "the maximum length of a line which is considered normal in most cases".

Reply:
`Outputs are Unicode strings, and can be a maximum of 1 MB. The total of all outputs in a workflow run can be a maximum of 50 MB.`

So .... we need this limitation, and `TEXT` doesn't work because in MySQL, you know ......

Reply:
Seems that the max length in GitHub is 1MB

> Outputs are Unicode strings, and can be a maximum of 1 MB. The total of all outputs in a workflow run can be a maximum of 50 MB.

See https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs#overview

Reply:
Done at d6db381c2e